apps/admin-gui/src/app/shared/components/dialogs/create-attribute-definition-dialog/create-attribute-definition-dialog.component.html

```diff
@@ -82,7 +82,9 @@ <h1 mat-dialog-title>{{'DIALOGS.CREATE_ATTRIBUTE_DEFINITION.TITLE' | translate}}
 
       <perun-web-apps-attribute-critical-operations-toggles
         (readOperationChanged)="finalReadOperations=$event"
-        (writeOperationChanged)="finalWriteOperations=$event">
+        (readGlobalChanged)="finalReadGlobal=$event"
+        (writeOperationChanged)="finalWriteOperations=$event"
```

Comment:
We need to change global criticality to false when changing this to false as well. Otherwise exception is thrown (see video)
[Screencast from 3.8.2023 13:26:21.webm](https://github.com/CESNET/perun-web-apps/assets/97669007/f962fe53-cf3e-4090-97f1-ccca5d7708ca)